nomad/server.go

```diff
@@ -1173,6 +1174,7 @@ func (s *Server) setupRpcServer(server *rpc.Server, ctx *RPCContext) {
 		s.staticEndpoints.ClientAllocations = &ClientAllocations{srv: s, logger: s.logger.Named("client_allocs")}
 		s.staticEndpoints.ClientAllocations.register()
 		s.staticEndpoints.ClientCSI = &ClientCSI{srv: s, logger: s.logger.Named("client_csi")}
+		s.staticEndpoints.ServiceRegistration = &ServiceRegistration{srv: s}
```

Comment:
I think this belongs up on L1170. Sadly this `// Client endpoints` section is a little ambiguous: they're endpoints called *on* clients, *not* by clients. Since we forward client RPCs through servers, the servers still need stub/proxy handlers for them installed, but the "real" logic is in `client/*_endpoint.go`.

Reply:
Ah yeah good catch; this is a mistake on my part.